### ISSUE TITLE
Export/import capabilities

### ISSUE THREAD

**Issue by subiol** [surtecha/karman#32]
It is great that Karman is all local, but that means there is no cloud to backup the users data, so an option to export and import the data of the app so it can be recreated in another phone would be great, even necessary.

**Comment by surtecha** [comment]
I agree that data backup is necessary, but it's not possible at the current stage. The app makes no money whatsoever and  would cost me money to store data on the servers. But not abandoning the idea.

**Comment by subiol** [comment]
I think I did not explain the request correctly. I am not asking for a cloud service that allows the data to be uploaded to. I understand running a server has a cost and was not requesting that.

Most apps that keep the data locally have an option to export the data into a file and import data from a file. This is the request. This way when, for example, changing phones, you can export the data to a file, load the file in the new phone and import it, keeping the app as it was in the previous phone.

**Comment by surtecha** [comment]
Hey! After lot of thought, I'm rebuilding the app from ground up. The app's architecture has been the roadblock all along. It had an inflexible design. If you're interested, you can join our community and share your ideas and suggestions to make it better.

The development has begun.
